src/neph/program.cr

```diff
@@ -9,6 +9,13 @@ class Program
     opt_parser.banner = "Usage: neph [options] [job name]"
     opt_parser.on "-v", "--version", "Show software version" { STDERR.puts VERSION; exit 0 }
     opt_parser.on "-h", "--help", "Show this help message" { STDERR.puts opt_parser; exit 0 }
+    opt_parser.on "-f", "--file", "Specify an alternative file" do |config|
+      unless File.exists?(config)
+        STDERR.puts "File does not exists"
+        exit 0
+      end
+      @filename = config
```

Comment:
`include` in config, `directory` in jobs, and all paths in `commands`  should be relative to the location of the **main** file. It should `cd` into the **dirname** of the argument, and set `@filename` to the **basename** of the argument.